pkg/qDSA/Makefile.dep

```diff
@@ -1,8 +1,8 @@
-ifneq (,$(filter cortex-m23 cortex-m0%,$(CPU_CORE)))
+ifneq (,$(filter arch_arm_cortex_m,$(FEATURES_USED)))
```

Comment:
Why not re-use the `cpu_core_cortexm` feature ? It seems to have the same purpose.
`

Reply:
Why do some arch features are prefixed `arch_` and others are `cpu_core_`? Maybe we should do an `s/cpu_core_cortexm/arch_arm_cortex_m/g`?

Reply:
Indeed but homogenizing this is another PR. In the context of this PR, there's no need to introduce another feature if there's already one that does the same job.

Reply:
In addition, I overlooked that the selection indeed depends on the specific Cortex M is build for. So the approach taken here doesn't work here anyway for the ARM selection.